super_sim.go

```diff
@@ -0,0 +1,233 @@
+	data := url.Values{}
+	data.Set("Iccid", iccid)
+	data.Set("RegistrationCode", registrationCode)
+	err := s.client.CreateResource(ctx, superSimPathPart, data, superSim)
+	return superSim, err
+}
+
+// Get finds a single SuperSim resource by its sid or unique name, or returns an error.
+func (s *SuperSimService) Get(ctx context.Context, sidOrUniqueName string) (*SuperSim, error) {
+	superSim := new(SuperSim)
+	err := s.client.GetResource(ctx, superSimPathPart, sidOrUniqueName, superSim)
+	return superSim, err
+}
+
+// Activate sets the status of the SIM provided to "active".
+//
+// See https://www.twilio.com/docs/iot/supersim/api/sim-resource#update-a-sim-resource
+func (s *SuperSimService) Activate(ctx context.Context, sid string) (*SuperSim, error) {
+	superSim := new(SuperSim)
+	data := url.Values{}
+	data.Set("Status", "active")
+	err := s.client.UpdateResource(ctx, superSimPathPart, sid, data, superSim)
+	return superSim, err
+}
+
+// Update updates the specified SuperSim resource with the data provided, or returns an error.
+func (s *SuperSimService) Update(ctx context.Context, sid string, data url.Values) (*SuperSim, error) {
+	superSim := new(SuperSim)
+	err := s.client.UpdateResource(ctx, superSimPathPart, sid, data, superSim)
+	return superSim, err
+}
+
+// GetPage returns a single Page of SuperSims, filtered by data.
+//
+// See https://www.twilio.com/docs/iot/supersim/api/sim-resource#read-multiple-sim-resources.
+func (s *SuperSimService) GetPage(ctx context.Context, data url.Values) (*SuperSimPage, error) {
+	return s.GetPageIterator(data).Next(ctx)
+}
+
+// GetPageIterator returns a superSimPageIterator with the given page
+// filters. Call iterator.Next() to get the first page of resources (and again
+// to retrieve subsequent pages).
+func (s *SuperSimService) GetPageIterator(data url.Values) *superSimPageIterator {
+	iter := NewPageIterator(s.client, data, superSimPathPart)
+	return &superSimPageIterator{
+		p: iter,
+	}
+}
+
+// Next returns the next page of resources. If there are no more resources,
+// NoMoreResults is returned.
+func (s *superSimPageIterator) Next(ctx context.Context) (*SuperSimPage, error) {
+	ap := new(SuperSimPage)
+	err := s.p.Next(ctx, ap)
+	if err != nil {
+		return nil, err
+	}
+	s.p.SetNextPageURI(ap.Meta.NextPageURL)
+	return ap, nil
+}
+
+type Fleet struct {
+	Sid                     string     `json:"sid"`
+	Url                     string     `json:"url"`
+	AccountSid              string     `json:"account_sid"`
+	UniqueName              string     `json:"unique_name""`
+	DataEnabled             bool       `json:"data_enabled"`
+	DataLimit               int        `json:"data_limit"`
+	DataMetering            string     `json:"data_metering"`
+	DateCreated             TwilioTime `json:"date_created"`
+	DateUpdated             TwilioTime `json:"date_updated"`
+	CommandsEnabled         bool       `json:"commands_enabled"`
+	CommandsUrl             url.URL    `json:"commands_url"`
```

Comment:
I don't think this will actually decode... I think this should be a string, people can parse to a url.URL later if they want.

(It was probably a mistake to accept a url.URL vs. just a string holding a URL elsewhere in this API, url.URL is kinda cumbersome to work with.)